prow/config.yaml

```diff
@@ -1239,6 +1240,7 @@ presubmits:
         - "--upload=gs://kubernetes-jenkins/pr-logs"
         - "--" # end bootstrap args, scenario args below
         - "--batch" # mitigation for https://github.com/bazelbuild/bazel/issues/4311
+        - "--test-args=--config=unit"
```

Comment:
looks like this needs to be moved down a few lines 🙃

Reply:
ahh copy paste is evil!
fixed